services/docker/docker-cloud-common-fetch.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const Joi = require('joi')
+
+const cloudBuildSchema = Joi.object({
+  objects: Joi.array()
+    .items(
+      Joi.object({
+        state: Joi.string(),
+        build_settings: Joi.array(),
+      }).required()
+    )
+    .required(),
+}).required()
+
+async function fetchBuild(serviceInstance, { user, repo }) {
+  return serviceInstance._requestJson({
+    schema: cloudBuildSchema,
+    url: `https://cloud.docker.com/api/build/v1/source/?image=${user}/${repo}`,
```

Comment:
let's use `qs` here just to ensure everything gets handled correctly, 

```suggestion
    url: `https://cloud.docker.com/api/build/v1/source`,
    options: { qs: { image: `${user}/${repo}` } },
```

Reply:
it breaks tests :cry:

Reply:
yeah I noticed that too, I left an inline comment on one of the testers as to what I think may be the cause

Reply:
👍 working now